rust/operator-binary/src/controller.rs

```diff
@@ -349,6 +359,39 @@ fn build_server_rolegroup_daemonset(
             ..Probe::default()
         })
         .build();
+
+    let container_bundle_helper = ContainerBuilder::new("opa-bundle-helper")
+        .image(format!(
+            "docker.stackable.tech/stackable/opa-bundle-helper:{}",
+            PKG_VERSION
+        ))
+        .command(vec![String::from("/stackable-opa-bundle-helper")])
+        .add_env_var_from_field_path("WATCH_NAMESPACE", FieldPathEnvVar::Namespace)
+        .add_volume_mount("bundles", "/bundles")
+        .build();
+
+    let init_container = ContainerBuilder::new("init-container")
+        .image("docker.stackable.tech/stackable/opa-bundle-helper:0.9.0-nightly")
+        .command(vec!["bash".to_string()])
+        .args(vec![
+            "-euo".to_string(),
+            "pipefail".to_string(),
+            "-x".to_string(),
+            "-c".to_string(),
+            [
+                "mkdir -p /bundles/active",
+                "mkdir -p /bundles/incomming",
+                "chown -R stackable:stackable /bundles/active",
+                "chown -R stackable:stackable /bundles/incomming",
+                "chmod -R a=,u=rwX /bundles/active",
+                "chmod -R a=,u=rwX /bundles/incomming",
+            ]
+            .join(" && "),
+        ])
+        .security_context(SecurityContextBuilder::run_as_root())
```

Comment:
Won't this mean that bundles that are written are owned by root and potentially not readable by the stackable user running opa if they inherit the directory permissions of  "u=rwX" ?

Reply:
This is the init container that sets the access properly for the `stackable` user. Both the helper and opa run as `stackable`.